.gitignore

```diff
@@ -10,3 +10,15 @@
 .externalNativeBuild
 gradle/wrapper/gradle-wrapper.properties
 gradle/wrapper/gradle-wrapper.jar
+.gradle/4.10.1/fileChanges/last-build.bin
```

Comment:
@Fafan2008 
Why do we need to ignore every file?
We can ignore the whole folder)